### PR TITLE
Renaming Batarangle to Augury in app

### PR DESCRIPTION
**Pull request by sumitarora**
Should fix: https://github.com/rangle/augury/issues/303

### CODE REVIEW

File: src/backend/adapters/angular2.ts [hunk 1]
```diff
@@ -121,7 +121,7 @@ export class Angular2Adapter extends BaseAdapter {
     const injectors = this._getComponentInjectors(debugEl, dependencies);
 
     description.unshift({
-      key: 'b-id',
+      key: 'a-id',
```
Comment (vanessayuenn) [inline src/backend/adapters/angular2.ts]: :+1: :smile_cat:

File: src/devtools/devtools.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
  * on Chrome Development Tools window.
 */
 chrome.devtools.panels.create(
-  'Angular2',
+  'Augury',
```
Comment (vanessayuenn) [inline src/devtools/devtools.ts]: do we want to keep this as `Angular 2`?

Comment (sumitarora) [inline src/devtools/devtools.ts]: I don't think so, this was one of the feedback in usability study also. @igor-ka ?

Comment (sumitarora) [inline src/devtools/devtools.ts]: @igor-ka Can you comment on this ?

**Comment by bertrandk** [comment]
lgtm

**Pull request merged**